sdks/js/src/types.ts

```diff
@@ -460,6 +460,24 @@ const BrowseActionTypeSchema = BaseActionSchema.extend({
 });
 type BrowseActionPublicType = z.infer<typeof BrowseActionTypeSchema>;
 
+const ConversationFileTypeSchema = z.object({
+  fileId: z.string(),
+  title: z.string(),
+  contentType: z.string(),
+});
+
+const ConversationListFilesActionTypeSchema = BaseActionSchema.extend({
```

Comment:
I'm surprised to see them in the public api types as it should not be needed to return them in the public json of the conversation, does it ?

Reply:
In practice they are never returned but I don't think it's necessarily an issue to have them leak because it's not entirely a lie.

Reply:
The next ConversationAgentActionType will be returned by the API and will live along this one so we're paving the way for them.

Reply:
The next ones yes, that one no.

Feels like we are mixing apple and oranges here.